### ISSUE TITLE
App crash

### ISSUE THREAD

**Issue by jaxxreal** [avocode/avocode#951]
Just tried to open application.

 v.2.8.0
Source UID: 67992
```
TypeError: Cannot read property 'completed_at' of null
    at t._handleAction (file:///home/wizards/%D0%97%D0%B0%D0%B3%D1%80%D1%83%D0%B7%D0%BA%D0%B8/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:19:8624)
    at Object.ID_13 (file:///home/wizards/%D0%97%D0%B0%D0%B3%D1%80%D1%83%D0%B7%D0%BA%D0%B8/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:19:3049)
    at Dispatcher._invokeCallback (/home/wizards/Загрузки/Avocode-linux-x64/resources/app.asar/node_modules/flux/lib/Dispatcher.js:198:24)
    at Dispatcher.dispatch (/home/wizards/Загрузки/Avocode-linux-x64/resources/app.asar/node_modules/flux/lib/Dispatcher.js:174:14)
    at e.dispatch (file:///home/wizards/%D0%97%D0%B0%D0%B3%D1%80%D1%83%D0%B7%D0%BA%D0%B8/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:21:8327)
    at t.e.dispatch (file:///home/wizards/%D0%97%D0%B0%D0%B3%D1%80%D1%83%D0%B7%D0%BA%D0%B8/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:2:20643)
    at t.assignItem (file:///home/wizards/%D0%97%D0%B0%D0%B3%D1%80%D1%83%D0%B7%D0%BA%D0%B8/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:15:12660)
    at e._addRemoteSlot (file:///home/wizards/%D0%97%D0%B0%D0%B3%D1%80%D1%83%D0%B7%D0%BA%D0%B8/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:14:711)
    at file:///home/wizards/%D0%97%D0%B0%D0%B3%D1%80%D1%83%D0%B7%D0%BA%D0%B8/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:13:31817
    at callback (file:///home/wizards/%D0%97%D0%B0%D0%B3%D1%80%D1%83%D0%B7%D0%BA%D0%B8/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:22:35)
    at t._handleResponse (file:///home/wizards/%D0%97%D0%B0%D0%B3%D1%80%D1%83%D0%B7%D0%BA%D0%B8/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:22:2649)
    at Request._callback (file:///home/wizards/%D0%97%D0%B0%D0%B3%D1%80
```

**Comment by connor-bean** [comment]
Hey, @jaxxreal -

Can you try downloading our newest version to see if that fixes this problem:

[Linux 64bit](https://avocode-media.s3.amazonaws.com/download/avocode-app/2.8.2/avocode-app-linux-2.8.2.zip)

[Linux 32bit](https://avocode-media.s3.amazonaws.com/download/avocode-app/2.8.2/avocode-app-linux-ia32-2.8.2.zip)

Please let us know if this fixes the problem.

**Comment by jakubzitny** [comment]
This was fixed in `2.8.1` already (#926). Closing.

**Comment by jaxxreal** [comment]
@connor-bean Thanks! I downloaded new version and it works fine.
